source/features/indented-code-wrapping.css

```diff
@@ -1,3 +1,8 @@
+.rgh-softwrapped-code .blob-code:not(.blob-code-hunk) {
+	line-height: 21px; /* Compensation for block-ifying table cells */
+}
+
+.rgh-code-wrapping-enabled .blob-code-hunk, /* Wrap code eagerly to avoid layout changes later */
```

Comment:
> * add a `rgh-softwrapped-code` class on `body` to enable regular wrapping on all elements before the rest of the javascript runs;

This should do the task, but I'm assuming this might affect fragment-targeted-link-loading (changing layout after page load might move the page over the desired view).